.github/workflows/main.yml

```diff
@@ -109,9 +121,55 @@ jobs:
     uses: ./.github/workflows/linux.yml
     with:
       wpt: 'test'
-      layout: ${{ fromJson(needs.decision.outputs.configuration).layout }}
+      wpt-layout: ${{ fromJson(needs.decision.outputs.configuration).linux_wpt_layout }}
       unit-tests: ${{ fromJson(needs.decision.outputs.configuration).unit_tests }}
 
+  report-test-results:
+    name: Report WPT Results
+    runs-on: ubuntu-latest
+    needs:
+      - "decision"
+      - "build-win"
+      - "build-mac"
+      - "build-linux"
+    if: ${{
+      always() && !cancelled() &&
```

Comment:
is `always()` needed here?

Reply:
Yes, I think it is. Without always, I think that dependent jobs failing or being cancelled can cause this job not to run. It's a weird quirk of GitHub Actions.

Reply:
Cool. Thanks for the background!